readthedocs/doc_builder/python_environments.py

```diff
@@ -374,7 +374,7 @@ def install_core_requirements(self):
                 self.project.get_feature_value(
                     Feature.DEFAULT_TO_MKDOCS_0_17_3,
                     positive='mkdocs==0.17.3',
-                    negative='mkdocs<1.1',
+                    negative='mkdocs',
```

Comment:
This may potentially break a lot of builds. We are currently installing 1.0.4 and this will install 1.1.2.

I agree with #7859 about installing its latest version, but we should schedule an end of supported date (as that policy suggest) for this change instead of just making it happen _now_.

Reply:
We can just use a feature flag.